pyanaconda/kickstart_dispatcher.py

```diff
@@ -0,0 +1,355 @@
+# Anaconda kickstart dispatching for modules
```

Comment:
Seemed like overkill to me at this point, but perhaps we should have "kickstart" directory (with parser, element subdirs) for this and at some point move all kickstart handling from kickstart.py there?

Reply:
That sounds great.

Reply:
hm, it would conflict with kickstart.py

Reply:
I would propose to just slightly change the name now to avoid conflict and rename it later when the kickstart.py will be moved in. What about `kickstart_handler` or `kickstart_dispatcher` for now?